app/src/main/java/com/codepath/apps/critterfinder/activities/FindActivity.java

```diff
@@ -31,15 +32,16 @@
 
 
 @RuntimePermissions
-public class FindActivity extends AppCompatActivity implements LocationService.OnLocationListener {
+public class FindActivity extends AppCompatActivity implements LocationService.OnLocationListener, PetSearch.PetSearchCallbackInterface {
```

Comment:
Shall we keep using FindActivity as a testbed for you to test the pet service and test searching?

I created PetBrowserActivity in setting up the skeleton for the tinder view including a stub fragment for the swipeable view.

Eventually the two activities should merge (I don't care which one we use) but it could make sense to have the two in parallel right now so you have a test bed to wire up search against without getting blocked.